options/locale/locale_en-US.ini

```diff
@@ -1414,6 +1414,7 @@ issues.action_check_all = Check/Uncheck all items
 issues.opened_by = opened %[1]s by <a href="%[2]s">%[3]s</a>
 pulls.merged_by = by <a href="%[2]s">%[3]s</a> was merged %[1]s
 pulls.merged_by_fake = by %[2]s was merged %[1]s
+pulls.changed_file_mode = %[1]s → %[2]s
```

Comment:
Would recommend https://primer.github.io/octicons/arrow-right-16 instead of this ugly ASCII arrow. Also, remove this translation. this text has nothing to do with language.

Reply:
I see GitHub uses the ASCII arrow too, but I still think the octicon is superior 😆 

<img width="154" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/98e01cbc-f740-4b25-9744-c15d1f2c1aa5">

Reply:
Agree, then no need to introduce a new translation string. And this translation string is untranslatable, the same for all languages.

Reply:
On the other hand we do use ASCII arrow as well for renames:

<img width="666" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/b4bee8a8-beee-43b2-b016-cff879189f48">

So I guess keep the ASCII arrow for consistency unless you want to change all places. Still would like to see the text removed from translation as it would translate the same in all languages.

Reply:
fixed in https://github.com/go-gitea/gitea/pull/24966/commits/cd3c58e1c7ec401abfa48d9763208eeb320277d5